app/views/pulls/_changes.html.erb

```diff
@@ -21,7 +21,7 @@
 
   <% diff = @pull.diff(@changes_from) %>
 
-  <% unless diff&.empty? %>
+  <% if diff && ! diff.empty? %>
```

Comment:
```suggestion
  <% unless diff&.empty? %>
```

Reply:
I've tested it, and it's not the same behaviour.